lib/document/templates.js

```diff
@@ -247,9 +263,35 @@ class PagedReadyHandler extends Paged.Handler {
     }
 }
 Paged.registerHandlers(PagedReadyHandler);
+
+window.PagedConfig = window.PagedConfig || {}
+
+window.PagedConfig.auto = false
+
+// same logic as pagedjs, but waiting for 'complete' instead of 'interactive'
+let ready = new Promise(function(resolve, reject){
+  if (document.readyState === "complete") {
+    resolve(document.readyState);
+    return;
+  }
+
+  document.onreadystatechange = function ($) {
```

Comment:
`$` is not used and can be removed right?

```suggestion
  document.onreadystatechange = function () {
```